source/chapters/configuration.rst

```diff
@@ -50,12 +53,39 @@ inputs and outputs to be used.
   input. The sample rate value should be set to the sample rate of your audio
   interface. By default, Mixxx tries the system default first, which is most
   likely 44.1 kHz. Otherwise, Mixxx will pick a different default.
-
+        
 * **Audio buffer**: Also known as latency, this is the lag time in milliseconds
   that it takes for Mixxx to process your input. Lower latency means Mixxx
   will be more responsive but on slower computers and cheaper sound cards it
   might cause glitches.
 
+* **Multi-Soundcard Synchronization**: Mixxx is able to use two or more soundcards 
+  at a time, each with its own clock. The Mixxx engine is driven by the Master 
+  soundcards. Here you can select the synchronization used for the other 
+  soundcards to avoid buffer overflows or underflows.
+
+* **Keylock/Pitch-Bending Engine**: Allows you to select the engine for all 
+  independent tempo and pitch changes like keylock. Use Soundtouch on old devices 
+  and netbooks or if you experience buffer underflows using :term:`key lock`.
+
+* **Master Mix**: You may disable the master mix to save some processing time, if 
+  you do not use the Master output, recording or live broadcasting.
+
+* **Master Output Mode**: In Mono Mode, the left and right channel are combined to a 
+  mono signal, which is passed to both channels of your master sound card. 
+  This is useful for outdoor setups where you can only hear the right or the left 
+  speaker. 
```

Comment:
Outdoor vs indoor isn't what's really important:
"This is useful in setups where the speakers are wired in mono (many clubs are setup this way) or there is only one speaker."